### ISSUE TITLE
Production Data Export Issue 

### ISSUE THREAD

**Issue by MightyAx** [ebi-ait/hca-ebi-wrangler-central#926]
TLDR: This was a permissions issue, Nate has fixed it!

[Monitoring Link](https://monitoring.ingest.archive.data.humancellatlas.org/goto/IbYn9-V4z)

- googleapiclient.errors.HttpError: 
- HttpError 400 when requesting https://storagetransfer.googleapis.com/v1/transferJobs?alt=json returned: 
    - **project-612961917858@storage-transfer-service.iam.gserviceaccount.com** does not have the required **pubsub.topics.publish** permission on topic
    - **projects/mystical-slate-284720/topics/broad-dsp-monster-hca.staging-transfer-notifications.ebi**

Impacts:
- Wrangling Ticket [#829](https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/829)
    - [Project](https://contribute.data.humancellatlas.org/projects/detail?uuid=dcbb50d1-9acf-4f70-9fda-b1f63a948c49&tab=upload)
    - [Submission](https://contribute.data.humancellatlas.org/submissions/detail?uuid=c68db894-cee7-4b22-bde3-5a4be4106fc4&project=dcbb50d1-9acf-4f70-9fda-b1f63a948c49)
    - [Export Job](https://api.ingest.archive.data.humancellatlas.org/browser/index.html#https://api.ingest.archive.data.humancellatlas.org/exportJobs/633ab5aa80fa451243cdd380)
    - Data Transferred / Metadata Transfered / Exported!
- Wrangling Ticket [#830](https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/830)
    - [Project](https://contribute.data.humancellatlas.org/projects/detail?uuid=c8e6c5d9-fcde-4845-bead-ff96999e3051&tab=upload)
    - [Submission](https://contribute.data.humancellatlas.org/submissions/detail?uuid=fd22294c-fcc5-468d-ad31-08e831513dd6)
    - [Export Job](https://api.ingest.archive.data.humancellatlas.org/browser/index.html#https://api.ingest.archive.data.humancellatlas.org/exportJobs/633ab83c80fa451243cdd381)
    - Data Transferred / Metadata Exporting / Exported!
- Wrangling Ticket **Not Found**
    - [Project](https://contribute.data.humancellatlas.org/projects/detail?uuid=2b81ecc4-6ee0-438f-8c5b-c10b2464069e&tab=upload)
    - [Submission](https://contribute.data.humancellatlas.org/submissions/detail?uuid=35ff1059-2a89-49b3-a861-95880fac5530)
    - [Export Job](https://api.ingest.archive.data.humancellatlas.org/browser/index.html#https://api.ingest.archive.data.humancellatlas.org/exportJobs/633b0c4980fa451243ce23d4)
    - Data Transferring

**Comment by MightyAx** [comment]
This seems like a permissions issue, have asked Nate for help.

**Comment by MightyAx** [comment]
This was a permissions issue since the new export code had never been run on production. Nate has fixed it!
